src/scripts/reducers/index.ts

```diff
@@ -11,8 +9,12 @@ import offlineReducer from "./offline/index";
 import downloadedDataReducer from "./downloaded-data";
 import welcomeScreenReducer from "./welcome-screen";
 import embedElementsReducer from "./embed-elements";
+import { layersApi, storiesApi } from "../services/api";
+import { combineReducers } from "@reduxjs/toolkit";
 
 const rootReducer = combineReducers({
+  [layersApi.reducerPath]: layersApi.reducer,
+  [storiesApi.reducerPath]: storiesApi.reducer,
```

Comment:
Why is that new here? Why to we need them in addition to `layersReducer` and `storiesReducer`?